libs/entry-components/styles/modules/vendors/redactor/_buttons-generator.scss

```diff
@@ -0,0 +1,22 @@
+@use 'sass:map';
+
+$material-button-background: #237BBB;
+
+@mixin generate-from($theme) {
+
+	$theming-buttons: map.get($theme, 'general', 'fonts', 'buttons');
+
+	.enigmatry-redactor-content button {
```

Comment:
This is a custom class that I added in Redactor lib, named enigmatry instead of entry to follow convention for internal packages, to be able to override Redactor default styling.